routers/default_router.py

```diff
@@ -16,19 +17,31 @@ def root(request: Request):
     }
 
 
-@router.post("/register/")
-def register(user: User):
-    if server.game.register_user(user):
-        return status.HTTP_200_OK
+@router.get("/register/{name}")
+def register(name: str, request: Request) -> StreamingResponse:
+    register_result = server.game.register_user(User(name=name), request)
+    if register_result is not False:
```

Comment:
Dobrze rozumiem, że rejestracja użytkownika polega na wysłaniu żądania pod dany adres, gdzie jeśli się uda to zwrócony zostanie strumień, a jeśli wystąpi błąd, to zwracamy status błędu?

Reply:
tak